src/helpers/data.js

```diff
@@ -105,31 +105,37 @@ export default {
       .map((datum) => {
         const x = accessor.x(datum);
         const y = accessor.y(datum);
-        return merge({}, datum, {
-          category: this.determineCategoryIndex(x, props.categories),
+        const category = this.determineCategoryIndex(x, props.categories);
+
+        return assign(
```

Comment:
We should be able to just use `Object.assign` and drop the lodash `assign` module.